CMakeLists.txt

```diff
@@ -30,6 +30,10 @@ set (rqt_gauges_SRCS
   src/rqt_gauges/my_plugin.cpp
 )
 
+set(rqt_qcgauge_HDRS
```

Comment:
Can you add a comment about why you need this, along with line 37? Something like (if I understand right):

"my_plugin.h is not necessary to export a C++ library"

Reply:
Will do!

Reply:
Actually, I think that the `qcgaugewidget` header doesn't need to be included in the `rqt_gauges_HDRS` variable, since the MOC-generated code is already included in the `qcgaugewidget` library - I'll try and clarify this w/ a comment.

Reply:
Thanks!